.github/workflows/build-brs.yml

```diff
@@ -21,13 +21,13 @@ jobs:
         with:
           node-version: 16.x
 
-      - name: Run simple
+      - name: Run e2e
         run: |
-          tasks/e2e-simple.sh
```

Comment:
Is this old script still required?

Reply:
Keep it but never run it, because of some good practice for us to write new scripts.

Reply:
Updated: have removed `e2e-installs.sh` and `e2e-simple.sh`.